lib/node-labels.js

```diff
@@ -78,6 +80,8 @@ const subSystemLabelsMap = new Map([
   [/^deps\/uvwasi\//, 'wasi'],
   [/^deps\/nghttp2\/nghttp2\.gyp/, ['build', 'http2', 'dont-land-on-v6.x']],
   [/^deps\/nghttp2\//, ['http2', 'dont-land-on-v6.x']],
+  [/^deps\/ngtcp2\//, ['quic'],
+  [/^deps\/nghttp3\//, ['quic'],
```

Comment:
Seeing the syntax error reported from Travis, could this be the culprit?

Looks like the ones above either has one more closing bracket or uses a plain `string`..